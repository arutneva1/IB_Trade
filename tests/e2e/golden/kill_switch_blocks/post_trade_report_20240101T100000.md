| symbol | side | filled_shares | avg_price | notional | residual_drift_bps |
| --- | --- | --- | --- | --- | --- |
| AAA | BUY | 99.0000 | 101.00 | 9999.00 | 44.30 |
