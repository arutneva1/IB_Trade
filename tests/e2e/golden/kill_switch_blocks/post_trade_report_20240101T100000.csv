symbol,side,filled_shares,avg_price,notional,residual_drift_bps
AAA,BUY,99.0,101.0,9999.0,44.3
